Svc/SeqDispatcher/test/ut/SeqDispatcherTester.cpp

```diff
@@ -0,0 +1,92 @@
+// ======================================================================
+// \title  SeqDispatcher.hpp
+// \author zimri.leisher
+// \brief  cpp file for SeqDispatcher test harness implementation class
+// ======================================================================
+
+#include "SeqDispatcherTester.hpp"
+
+namespace Svc{
+
+// ----------------------------------------------------------------------
+// Construction and destruction
+// ----------------------------------------------------------------------
+
+SeqDispatcherTester ::SeqDispatcherTester()
+    : SeqDispatcherGTestBase("SeqDispatcherTester", SeqDispatcherTester::MAX_HISTORY_SIZE),
+      component("SeqDispatcher") {
+  this->connectPorts();
+  this->initComponents();
+}
+
+SeqDispatcherTester ::~SeqDispatcherTester() {}
+
+// ----------------------------------------------------------------------
+// Tests
+// ----------------------------------------------------------------------
+
+void SeqDispatcherTester ::testDispatch() {
```

Comment:
These are pretty basic. It would be nice to really nuke this with the random rules-based tester.

Reply:
Would love to do this! Is there an example of that in the code I can look at?

Reply:
@zimri-leisher I'll respond since I see your comment and I also have been learning about the rule-based tester recently.

Here is the source / README : https://github.com/nasa/fprime/tree/devel/STest

If you search/grep for "STest" across the codebase you'll find a bunch of modules and components that use it. `STest::RandomScenario` is the random one.

This makes me realize, we really should write a HowTo guide on how to use this...

Reply:
Here is an example: https://github.com/nasa/fprime/tree/devel/Svc/DpManager/test/ut

Reply:
This could also be a future work issue.

Reply:
Yeah, in the interest of getting this merged, I think I will just increase the coverage of the normal unit tests.